saves/options.xml

```diff
@@ -1,5 +1,5 @@
 <options>
-    <language>en</language>
+    <language>zh_cn</language>
```

Comment:
This shouldn't have been change.

Default language should stay to English, could you revert this change please?